src/main/java/com/mraof/minestuck/data/dialogue/ConsortDialogue.java

```diff
@@ -668,6 +685,14 @@ private static void consortDialogues(SelectableDialogueProvider provider, Dialog
 				new NodeBuilder(l.defaultKeyMsg("Look I know you aren't a reptile, I just got one question. You ever hear some of these people talk? I swear, they must be putting mercury in the water with how nonsensical conversation can be with these guys.")));
 		provider.addRandomlySelectable("hats", defaultWeight(isAnyEntityType(SALAMANDER)),
 				new NodeBuilder(l.defaultKeyMsg("I like crumpled hats, they're comfy and easy to wear!")).animation(HAPPY_EMOTION));
+		provider.addRandomlySelectable("best_scales", defaultWeight(isAnyEntityType(NAKAGATOR, IGUANA)),
+				new NodeBuilder(l.defaultKeyMsg("Everyone loves me for my luxurious shiny scales and cool demeanour.")));
+		provider.addRandomlySelectable("laser_defense_system", defaultWeight(isAnyEntityType(TURTLE)), new ChainBuilder()
+				.node(new NodeBuilder(l.defaultKeyMsg("Anytime someone approaches me when I want to be left alone I picture an orbital defense satellite beaming a laser down at them with precise precision.")))
+				.node(new NodeBuilder(l.defaultKeyMsg("I'm exploding you with my mind right now,,"))));
```

Comment:
Are the commas intentional?

Reply:
The commas are intentional. Although I only used two of them, they are meant to be a shitpost-esque equivalent to an ellipsis. 

https://english.stackexchange.com/questions/230189/origin-of-the-triple-comma-or-comma-ellipsis